microcosm_pubsub/chain/statements/call.py

```diff
@@ -0,0 +1,51 @@
+"""
+Syntactic sugar using existing primitives defined
+
+"""
+from microcosm_pubsub.chain import Chain
+from microcosm_pubsub.chain.context import ScopedSafeContext
+
+
+class LocalCallWrapper:
```

Comment:
The approach taken here is somewhat similar to what we do with `for_each`. We create a sub-chain with a scoped context.
My first approach was using a combination of `binds` and `assign_function`; but `binds` doesn't allow us to do the same operation twice on the same function. Besides, the end result is much clearer this way.